roles/update_containers/defaults/main.yml

```diff
@@ -19,7 +19,6 @@
 # All variables within this role should have a prefix of "cifmw_update_containers"
 
 cifmw_update_containers: false
-cifmw_update_containers_metadata: controlplane
```

Comment:
We'll need to add this variable to the molecule `converge.yml` to keep molecule tests passing:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/update_containers/molecule/default/converge.yml#L20

Reply:
This will fix the molecule failure at least:
https://review.rdoproject.org/zuul/build/39aa4e655e3b408faba38a20ab08c34c

Looking into the Adoption failure now